From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Kevin Hilman <khilman@baylibre.com>
Date: Thu, 25 Feb 2021 11:37:41 -0800
Subject: ANDROID: GKI: amlogic: enable GPIO LEDs as module

Enable GPIO LEDs as module for use on Amlogic boards such as Khadas
VIM3 family.

Bug: 179406580
Change-Id: I0952970ae3b7c2c1d6c09ca4a405c2815caf92ae
Signed-off-by: Kevin Hilman <khilman@baylibre.com>
---
 arch/arm64/configs/amlogic_gki.fragment | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm64/configs/amlogic_gki.fragment b/arch/arm64/configs/amlogic_gki.fragment
--- a/arch/arm64/configs/amlogic_gki.fragment
+++ b/arch/arm64/configs/amlogic_gki.fragment
@@ -6,6 +6,7 @@ CONFIG_REGULATOR_PWM=m
 CONFIG_PWRSEQ_EMMC=m
 CONFIG_PWRSEQ_SIMPLE=m
 CONFIG_USB_DWC2=m
+CONFIG_LEDS_GPIO=m
 
 #
 # Networking
